CONTRIBUTING.rst

```diff
@@ -472,6 +472,53 @@ This effectively invokes ``SPHINXOPTS=-j`` and can be especially useful for
 multi-core computers when ``FULL_DOC_BUILD=TRUE``
 
 
+
+Contributing to the Documentation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Documentation for PyVista is generated from three sources:
+
+- Docstrings from the classes, functions, and modules of ``pyvista`` using
+  `sphinx.ext.autodoc
+  <https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html>`_.
+- Restructured test from ``doc/``
+- Gallery examples from ``examples/``
+
+General usage and API descriptions should be placed within ``doc/api`` and
+the docstrings.  Full gallery examples should be placed in ``examples``.
+
+
+Adding a New Example
+^^^^^^^^^^^^^^^^^^^^
+PyVista's examples come in two formats: basic code snippets demonstrating the
+functionality of an individual method, or a full gallery example displaying one
+or more concepts.  Small code samples and snippets are contained in the
+``doc/api`` directory or within our documentation strings, while the full
+gallery examples, meant to be run as individual downloadable scripts, are
+contained in the ``examples`` directory at the root of this repository.
+
+To add a fully fledged, standalone example, add your example to the
+``examples`` directory within one of the applicable subdirectories.  Should
```

Comment:
We may want to be more clear that this is the `examples/` directory at the root level of the repository and not the subpackage within `pyvista`